db/schema.rb

```diff
@@ -301,6 +301,8 @@
     t.string "lionpath_semester"
     t.string "academic_program"
     t.string "degree_checkout_status"
+    t.datetime "author_release_warning_sent_at"
```

Comment:
This should probably be removed before we merge this to main so the migrations don't get mixed up